ether/core.py

```diff
@@ -26,8 +29,25 @@ class Connection(NamedTuple):
     source: NetworkNode
     target: NetworkNode
     latency: float = 0
+    latency_dist: ParameterizedDistribution = None
     # TODO: better network QoS modeling
 
+    def get_latency(self) -> float:
+        if self.latency_dist:
+            return self.latency_dist.sample()
+        return self.latency
+
+    def get_mode_latency(self) -> float:
+        if self.latency_dist:
+            dist = self.latency_dist
+            return np.exp(np.log(dist.scale) - dist.args[0] ** 2) + dist.loc
```

Comment:
maybe add a comment that this is only works correctly with lognormal distributions